lib/wrap-request.js

```diff
@@ -0,0 +1,59 @@
+/* eslint padded-blocks: 0 */
```

Comment:
If possible, I’d like to avoid any linting customizations. I agree that the code is more readable that way, but in the end I just don’t want to think about linting any more and want to avoid discussions about it with future contributors at all costs :D

Reply:
Agreed. I tried to format it the way the linter asked me to, but it was unreadable, so I decided to make a small exception... let's keep it a secret! 😄